.gitignore

```diff
@@ -8,4 +8,5 @@ reports
 
 # Dependency directories
 node_modules/
-package-lock.json
\ No newline at end of file
+package-lock.json
+.vscode
```

Comment:
Why are you adding .vscode here? I want this to be checked in so that each developer on team can follow same editor features and configurations.

Reply:
resoved.